<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="INWS03011-02" ignored="0" name="Test" tests="4" failures="1" timestamp="02 Jun 2021 12:33:45 GMT" time="0.058" errors="0">
  <testcase name="Test" time="0.006" classname="com.project.test.Testcases.TestAnnotation"/>
  <testcase name="Test1" time="0.001" classname="com.project.test.Testcases.TestAnnotation"/>
  <testcase name="Test" time="0.001" classname="com.project.test.Testcases.TestAnnotation2"/>
  <testcase name="Test1" time="0.002" classname="com.project.test.Testcases.TestAnnotation2">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.project.test.Testcases.TestAnnotation2.Test1(TestAnnotation2.java:22)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
</testsuite> <!-- Test -->
